packages/plugin-flex/src/commands/flex/plugins/start.ts

```diff
@@ -14,6 +14,11 @@ delete baseFlags.json;
 
 const MULTI_PLUGINS_PILOT = FLAG_MULTI_PLUGINS.substring(2);
 
+enum FlexPluginStartFlag {
```

Comment:
So this is good, but maybe revert it for now - if we are doing flag enums, then we should do it for ALL commands. Revert this for now and we will do a separate PR and move all of these as enums